include/cute/atom/copy_traits_xe.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <cute/atom/copy_traits.hpp>
+#include <cute/atom/copy_atom.hpp>
+
+#include <cute/arch/copy_xe.hpp>
+
+namespace cute
+{
+    template <class GTensor>
+    struct Copy_Traits<XE_2D_LOAD, GTensor>
+    {
+        // using ThrID   = Layout<_16>; //TODO: I think it should be 16 (copy is per subgroup) - but static_assert fails
+        using ThrID = Layout<_1>;
+        using NumBits = Int<sizeof(typename GTensor::engine_type::value_type) * 8>; // hacky: does vec of 8
+        // Map from (src-thr,src-val) to bit
+        using SrcLayout = Layout<Shape<_1, NumBits>>; // TODO:  is _1 correct?
+        // Map from (dst-thr,dst-val) to bit
+        using DstLayout = Layout<Shape<_1, NumBits>>;
+        // Reference map from (thr,val) to bit
+        using RefLayout = SrcLayout;
+
+        GTensor tensor;
+
+        template <class TS, class SLayout,
+                  class TD, class DLayout>
+        CUTE_HOST_DEVICE friend constexpr void
+        copy_unpack(Copy_Traits const &traits,
+                    Tensor<ViewEngine<ArithmeticTupleIterator<TS>>, SLayout> const &src,
+                    Tensor<TD, DLayout> &dst)
+        {
+            static_assert(is_rmem<TD>::value);
+            int H = size<0>(traits.tensor);
+            // int W = size<1>(traits.tensor) * sizeof(typename decltype(traits.tensor)::engine_type::value_type);
+            int W = size<1>(traits.tensor) * sizeof(typename TD::value_type); //TODO: inconsistent to give the size in elements but use vector for copy
+            auto [y, x] = src.data().coord_;
+            XE_2D_LOAD::copy(traits.tensor.data().get(), W, H, W, int2_{x, y}, &*dst.data());
```

Comment:
```suggestion
            XE_2D_LOAD::copy(traits.tensor.data().get(), W, H, W, int2_{static_cast<int>(x), static_cast<int>(y)}, &*dst.data());
```

Reply:
What's the motivation for this suggestion? Why do you want to allow narrowing conversion?

Reply:
I am getting compile time errors when integrating this code into the PVC pipeline without the conversion.

Reply:
What tensor do you pass to the copy? What error do you get?

Reply:
Can we resolve this? Maybe by merging as-is and open an issue to discuss this. Otherwise we have the other PRs depending on this having to make changes later.

Reply:
I hope you don't mind I merged it without resolving this. Needed to clean up the PR dependency issue. Please open a new PR with your suggested change if you still think it is needed.